File: php/examples/searchSeries.php
<?php

$search = 'Dragonball';
if(!empty($_GET['s'])) {
	$search = $_GET['s'];
}

require_once '../src/BurningSeries.php';
$bs = new BurningSeries();

// If the second parameter is set to "true" only exact matches will be returned
$series = $bs->search($search);

// Should be escaped normally but this is only an example
echo "Search for {$search} gave back: <br />";

foreach($series as $serie) {
	echo "{$serie['series']} (ID: {$serie['id']}) <br />";
}

File: java/examples/de/jonesboard/burningseries/examples/searchSeries.java
package de.jonesboard.burningseries.examples;

import de.jonesboard.burningseries.BurningSeries;
import de.jonesboard.burningseries.interfaces.SerieInterface;

public class searchSeries {

	public static void main(String[] args) {
		String search = "Dragonball";

		BurningSeries bs = new BurningSeries();

		// If the second parameter is set to "true" only exact matches will be returned
		SerieInterface[] series = bs.search(search);

		System.out.println("Search for " + search + " gave back:");
		
		for(SerieInterface serie : series) {
			System.out.println(serie.getName() + " (ID: " + serie.getId() + ")");
		}
	}


}
